app/src/main/java/com/bakkenbaeck/sol/BaseApplication.kt

```diff
@@ -0,0 +1,133 @@
+package com.bakkenbaeck.sol
+
+import android.app.AlarmManager
+import android.app.Application
+import android.app.PendingIntent
+import android.content.Intent
+import android.graphics.BitmapFactory
+import android.support.v4.app.NotificationCompat
+import android.text.Html
+import android.util.Log
+import com.bakkenbaeck.sol.extension.getAlarmService
+import com.bakkenbaeck.sol.extension.getNotificationService
+import com.bakkenbaeck.sol.repository.LocationRepository
+import com.bakkenbaeck.sol.service.AlarmReceiver
+import com.bakkenbaeck.sol.util.SolPreferences
+import com.bakkenbaeck.sol.view.SunActivity
+
+class BaseApplication : Application() {
+
+    companion object {
+        const val ACTION_UPDATE = "com.bakkenbaeck.sol.UPDATE"
+        const val EXTRA_DAILY_MESSAGE = "daily_message"
+        const val EXTRA_SUNRISE_TIME = "sunrise_time"
+        const val EXTRA_SUNSET_TIME = "sunset_time"
+        const val EXTRA_LOCATION_MESSAGE = "location_message"
+        const val EXTRA_CURRENT_PHASE = "current_phase"
+
+        lateinit var instance: BaseApplication
+    }
+
+    lateinit var locationRepository: LocationRepository
+
+    init {
+        instance = this
+    }
+
+    override fun onCreate() {
+        super.onCreate()
+        init()
+    }
+
+    private fun init() {
+        initLocationRepository()
+        initCalligraphy()
+        initObservers()
+    }
+
+    private fun initLocationRepository() {
+        locationRepository = LocationRepository(applicationContext)
+    }
+
+    private fun initCalligraphy() {
+        try {
+//            CalligraphyConfig.initDefault(CalligraphyConfig.Builder()
+//                    .setDefaultFontPath("fonts/gtamericalight.ttf")
+//                    .setFontAttrId(R.attr.fontPath)
+//                    .build()
+//            )
```

Comment:
I guess you commented this out because you don't have the font file, and I know this is WIP, but be careful about checking in commented out code. It has a habit of not getting uncommented out.